CHANGELOG.md

```diff
@@ -49,6 +49,9 @@ was subsequently created to document the work being done.
 
 ## Notable fixes and improvements
 
+- `declare` has been added as an alias for `typeset` (issue #220).
+- `local` has been added as a contrainsed alias for `typeset` when used inside
```

Comment:
Looks like a typo for "constrained".  I wouldn't matter that much except that it's in the publicy visible change log.